content/faqs.md

```diff
@@ -0,0 +1,67 @@
+
+---
+title: "Frequently asked questions"
+---
+
+
+## What is inclusive naming about?
+
+The software industry sometimes uses certain words, like blacklist, and slave, that may carry a great deal of emotional and historical baggage. We are encouraging projects to choose phrases that clearly communicate the technical meaning, without using metaphors or colloquialisms.
+
+This has the dual purpose of removing problematic terms, and also communicating more clearly, particularly to those who are working in a secondary language.
+
+Our goals are to identify where problematic terms are used, collaborate on replacement terminology, support project contributors in making updates, and report on joint progress.
+
+## Why is this important?
+
+If software is truly meant to be inclusive and a place where anyone can participate, it must be welcoming to all. If words or phrases convey secondary unintended meanings to our audience (or are simply confusing!) we are potentially limiting participation in our projects, which is antithetical to this goal.
+
+## What terms are we recommending you consider?
+
+We have identified the terms "master/slave" and "whitelist/blacklist" to work on because these are considered most visible and problematic across the industry.
+
+Over time, we may recommend consideration of other terms, such as words that reference mental health, gender, physical handicaps, and several other categories. We may also give tips on avoiding colloquialisms that simply don't translate well or prove a barrier to understanding.
+
+## What are the recommended replacement terms?
+
+See the word replacement list for all terms we recommend replacing.
```

Comment:
is there a link for this?

Reply:
It's in another PR – there's a bit of cleanup to do after all these merge.